### ISSUE TITLE
Retina 2X Issue

### ISSUE THREAD

**Issue by nickff** [modxcms/Resizer#17]
Hello,

First off, great plugin.

Secondly, I'm looking to do something slightly different than what Resizer does normally, and wanted to get your thoughts to see if it's possible. What I'm wanting is to be able to upload images at 2X their display size (3200px wide), keep that natural width on retina devices (though only displaying at 1600), **but then reduce the natural width dynamically to 1600px for non-retina devices**.

Basically, I'm trying to avoid having to make two versions of every image, one 2X for retina and one at display size. Is this possible with Resizer? Let me know!

**Comment by jonleverrier** [comment]
did you find a way to achieve this @nickff ?

**Comment by sepiariver** [comment]
`srcset` with `2x` flag is the best way I've found to achieve this.

```
<img src="[[+full_size_image]]" srcset="[[+full_size_image]] 2x, [[+full_size_image:pthumb=`w=1600`]] 1x">
```
This is handled by modern browsers and doesn't depend on pThumb nor Resizer specifically.
